applications/search/src/pages/api-details-page/resolved-api-details-page.jsx

```diff
@@ -20,6 +24,18 @@ const mapProps = {
     );
     const result = await Promise.all(promiseMap);
     return result;
+  },
+  referencedInformationModels: async props => {
+    const getApi = await memoizedGetApi(props.match.params.id);
```

Comment:
Du kan rename 'getApi' til det som du faktisk får, f.eks. 'apiItem'.

Reply:
Fixed in newest commit